bucket/dbvis.json

```diff
@@ -1,6 +1,8 @@
 {
-    "homepage": "https://www.dbvis.com/",
     "version": "10.0.21",
+    "homepage": "https://www.dbvis.com/",
+    "description": "A universal database tool for developers, DBAs and analysts.",
+    "license": "Proprietary",
```

Comment:
No url for proprietary license

Reply:
Could you tell me what do `Proprietary` and `Freeware` mean? I can't find an explanation in doc.

Reply:
https://en.wikipedia.org/wiki/Freeware

> There is no agreed-upon set of rights, license, or EULA that defines freeware unambiguously

https://en.wikipedia.org/wiki/Software_license#Proprietary_software_licenses

> The hallmark of proprietary software licenses is that the software publisher grants the use of one or more copies of software under the end-user license agreement (EULA), but ownership of those copies remains with the software publisher (hence use of the term "proprietary"). This feature of proprietary software licenses means that certain rights regarding the software are reserved by the software publisher. Therefore, it is typical of EULAs to include terms which define the uses of the software, such as the number of installations allowed or the terms of distribution.

Reply:
I got. Freeware does not means free in freedom. Thanks!